pages/agent-economies/mech.jsx

```diff
@@ -1,12 +1,18 @@
 import PageWrapper from 'components/Layout/PageWrapper';
+import { MechAgentMetrics } from 'components/MechPage/MechAgentMetrics';
 import { MechHero } from 'components/MechPage/MechHero';
+import { WhatIsOlasMech } from 'components/MechPage/WhatIsOlasMech';
+import { WhyOlasMech } from 'components/MechPage/WhyOlasMech';
 import Meta from 'components/Meta';
 
 const Mech = () => (
   <PageWrapper>
     <Meta pageTitle="Mech" description="AI intelligence for agent economies" />
```

Comment:
`intelligence` is redundant as mentioned above..
would go for: 
- services
- marketplace
etc ...
```suggestion
    <Meta pageTitle="Mech" description="AI services for agent economies" />
```